flink-runtime/src/main/java/org/apache/flink/runtime/io/network/buffer/NetworkBufferPool.java

```diff
@@ -274,7 +260,7 @@ private void redistributeBuffers() throws IOException {
 			return;
 		}
 
-		/**
```

Comment:
Actually, this was intentional since this was marked as a "dangling javadoc" by IntelliJ (it is just a longer inline-comment). I don't have too strong feelings about it though so we could either keep it or revert it.